src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  }
+
+  return days * 40 - 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T18%3A34%3A59Z) 1st item. You should create variables for main values - 20, 50, 40, 3, 7 they shouldn't be just magic numbers

Reply:
Okey